backend/package.json

```diff
@@ -53,6 +53,7 @@
     "@types/express-session": "^1.17.4",
     "@types/jest": "27.0.2",
     "@types/node": "^16.0.0",
+    "@types/passport-github2": "^1.2.5",
```

Comment:
@josayko Don't know why, after pulling from the `dev` branch from the main repo onto my branch, couldn't run backend, was missing passport-github2. Needed to install this package with `--save-dev` option